diff --git a/src/Game/main.cpp b/src/Game/main.cpp
index 99c4d8a1..7d929163 100644
--- a/src/Game/main.cpp
+++ b/src/Game/main.cpp
@@ -240,7 +240,7 @@ static void Initialize()
     AIPadManager::Startup();
     FEInput::Initialize();
     FlickDetection::Initialize();
-    CARDInit();
+    CARDInit("G4QE", "01");
     MemCard::s_InitDone = true;
 
     g_pTheLoadingManagerTask = new (nlMalloc(sizeof(LoadingManager), 8, false)) LoadingManager(0x14);
@@ -460,7 +460,7 @@ static void DoMemCheck()
 /**
  * Offset/Address/Size: 0x0 | 0x80173510 | size: 0x224
  */
-int main()
+int game_main()
 {
     if (g_DoStackWatermarkTests)
     {
@@ -475,12 +475,12 @@ int main()
 
     if (GetConfigBool(Config::Global(), "enableFloatingPointExceptions", false))
     {
-        InstallFloatingPointExceptionHandler();
+        // InstallFloatingPointExceptionHandler();
     }
 
     if (GetConfigBool(Config::Global(), "callStackDumper", false))
     {
-        InstallCallStackDumper();
+        // InstallCallStackDumper();
     }
 
     for (;;)
